channeldb/channel.go

```diff
@@ -2108,8 +2131,16 @@ func (c *OpenChannel) ActiveHtlcs() []HTLC {
 	// as active if *we* know them as well.
 	activeHtlcs := make([]HTLC, 0, len(remoteHtlcs))
 	for _, htlc := range c.LocalCommitment.Htlcs {
+		log.Debugf("LocalCommitment has htlc: id=%v, update=%v "+
+			"incoming=%v", htlc.HtlcIndex, htlc.LogIndex,
+			htlc.Incoming)
+
 		onionHash := sha256.Sum256(htlc.OnionBlob[:])
 		if _, ok := remoteHtlcs[onionHash]; !ok {
+			log.Debugf("Skipped htlc due to onion has not "+
+				"matched: id=%v, update=%v incoming=%v",
```

Comment:
strange wording. Perhaps "Skipped htlc since the onion does not match" or "Skipped htlc due to onion mismatch"?

Reply:
fixed!